File: test-1-simple-rest-api/spring/spring-sample/src/main/java/com/peacecwz/springsample/models/GreetingModel.java
package com.peacecwz.springsample.models;

public class GreetingModel {

    private final long id;
    private final String content;

    public GreetingModel(long id, String content) {
        this.id = id;
        this.content = content;
    }

    public long getId() {
        return id;
    }

    public String getContent() {
        return content;
    }
}

File: test-1-simple-rest-api/net-core/AspNetCoreApp/src/AspNetCoreApp.API/Models/GreetingModel.cs
namespace AspNetCoreApp.API.Models
{
    public class GreetingModel
    {
        public GreetingModel(int id, string name)
        {
            Id = id;
            Name = name;
        }
        
        public int Id { get; set; }
        public string Name { get; set; }
    }
}